### DISCUSSION TITLE
Keyboard Backlight Reports Vantage Running When It's Not

### DISCUSSION THREAD

**Discussion by angatia** [BartoszCichecki/LenovoLegionToolkit#852] [Q&A] [answered]
Since the last update before 2.14.3 the keyboard backlight control worked - I even had Vantage uninstalled. After the update I saw the backlight off and had this message that Lenovo Vantage seems to be enabled. I couldn't find any Lenovo service running. Anyone with the same issue?

Legion 5 Pro 16IAH7H
Win 11 Home 22H2

**Comment by BartoszCichecki** [comment, marked as answer]
If it reports that Vantage is running, then it's running. Check FAQ in readme for more details.

**Reply by Ahmedhossamdev** [reply]
same here
